src/submesh.cpp

```diff
@@ -385,15 +389,23 @@ namespace OgreMayaExporter
 	// Write submesh data to an Ogre compatible mesh
 	MStatus Submesh::createOgreSubmesh(Ogre::MeshPtr pMesh,const ParamList& params)
 	{
+		Ogre::MaterialManager* matMgr = Ogre::MaterialManager::getSingletonPtr();
 		MStatus stat;
 		// Create a new submesh
 		Ogre::SubMesh* pSubmesh;
 		if (m_name != "")
 			pSubmesh = pMesh->createSubMesh(m_name.asChar());
 		else
 			pSubmesh = pMesh->createSubMesh();
+        const Ogre::String materialName = m_pMaterial->name().asChar();
```

Comment:
Whoa. It looks like here on down the indentation got all jacked up, unless that's just how it reads in this review.

Reply:
It looks okay to me.